app/client/ui/UserManager.ts

```diff
@@ -103,13 +106,13 @@ export function showUserManagerModal(userApi: UserAPI, options: IUserManagerOpti
     if (model.isSelfRemoved.get()) {
       const name = resourceName(model.resourceType);
       confirmModal(
-        `You are about to remove your own access to this ${name}`,
-        'Remove my access', tryToSaveChanges,
+        t(`You are about to remove your own access to this {{name}}`, { name }),
```

Comment:
Renaming this from "name" to "resourceType" would be clearer I think. It may have values like "document", "workspace", or "team site".